src/modules/Builder/components/RemovePhrasePopup/RemovePhrasePopup.tsx

```diff
@@ -0,0 +1,31 @@
+import { Button } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+
+import { Modal, ModalProps } from 'shared/components';
+import { StyledFlexAllCenter, StyledFlexTopStart } from 'shared/styles';
+
+export const RemovePhrasePopup = ({
+  onRemove,
+  ...otherProps
+}: Omit<ModalProps, 'title' | 'children'> & { onRemove?: () => void }) => {
+  const { t } = useTranslation('app');
+
+  return (
+    <Modal
+      footer={
+        <StyledFlexAllCenter sx={{ width: '100%' }}>
+          <Button color="error" onClick={onRemove} variant="contained">
+            {t('phrasalTemplateRemovePopup.confirm')}
+          </Button>
+        </StyledFlexAllCenter>
+      }
+      hasActions={false}
```

Comment:
It's not necessary to do a custom footer – the base props centre-align the only button by default:

```suggestion
      onSubmit={onRemove}
      buttonText={t('phrasalTemplateRemovePopup.confirm')}
      submitBtnColor="error"
```